src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -56,6 +58,10 @@ const EmbeddingContinuousPlot = () => {
   );
   const expressionError = useSelector((state) => state.genes.expression.error);
   const { data, loading, error } = useSelector((state) => state.embeddings[embeddingType]) || {};
+  const cellSets = useSelector((state) => state.cellSets);
+  const {
+    loading2, error2, properties,
```

Comment:
`loading2` and `error2` are different from `loading` and `error`. Can you change these variable names to make it easier to understand?